nix/dev/scripts.nix

```
@@ -120,6 +124,21 @@ let
       --verbosity 2
   '';
 
+  start-marlowe-run = writeShellScriptBinInRepoRoot "start-marlowe-run" ''
+    #!/bin/bash
+    tmux -T 256,mouse,focus,title\
```

It would be better to use `tmux` from nixpkgs instead of the user's natively installed `tmux` (which they may not have installed):

```suggestion
   ${pkgs.tmux}/bin/tmux -T 256,mouse,focus,title\
```